flash/data/transforms.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any, Mapping, Sequence, Union
+
+from torch import nn
+
+from flash.data.utils import convert_to_modules
+
+
+class ApplyToKeys(nn.Sequential):
```

Comment:
this will work only for single inputs. E.g. in segmentation or tasks that require augmenting not only the inputs we still need something like this 
https://github.com/PyTorchLightning/lightning-flash/pull/239/files#diff-10dfb9297339fe1066151bc80b43feae338b80bfc185f47b951530e388b68719R38

Reply:
It's tricky, this class allows multi-input by extracting mutliple keys to then be passed to a single aug. So if you had `my_augmentation(image=..., mask=...)` then this way will work. It's not quite right as it assumes tuple inputs (which won't work with albumenmtations) and we will need something different again for Kornia haha. So, overall, we need something smarter here, but I don't know what yet

Reply:
agree, my only point was that to apply the same transformation to different inputs we might need to hold the generated random parameters.